x/common/constants.go

```diff
@@ -3,5 +3,5 @@ package common
 const (
 	TreasuryPoolModuleAccount = "treasury_pool"
 	// Precision for int representation in sdk.Int objects
-	Precision = int64(1_000_000)
+	MICRO = int64(1_000_000)
```

Comment:
`MICRO` or `MEGA`? Also what's wrong with `Precision`?

In my mind, 
micro ==> multiply by 10^-6
mega ==> multiply by 10^6

Reply:
If you have 2 dollars, you have 2 million micro dollars. Micro is multiplication.

`Precision` is vague, and someone wouldn't know what it means by looking at it. In our case, we're not really using this value as a precision. It's just a simple multiplier into micro units.

Reply:
Yes, `TO_MICRO` or `AS_MICRO` would be more clear.